future/src/ct/ct_imports.cc

```diff
@@ -1175,7 +1026,7 @@ void CtZimImport::_ensure_notebook_file_in_dir(const fs::path& dir)
     }
 }
 
-
+CtMDImport::~CtMDImport() = default;
```

Comment:
why destructor in .cc file?

Reply:
`CtMDParser` is forward declared and then used via an `std::unique_ptr`, the default generated dtor doesn't work because it is inline and `std::unique_ptr`'s dtor needs to have access to the complete type